go/inst/instance_topology.go

```diff
@@ -1675,6 +1677,33 @@ func TakeSiblings(instanceKey *InstanceKey) (instance *Instance, takenSiblings i
 	return instance, len(relocatedReplicas), err
 }
 
+// Created this function to allow a hook to be called after a successful TakeMaster event
+func TakeMasterHook(master *Instance, slave *Instance) {
+	masterInstance := master.Key
```

Comment:
please name this `masterKey`. a `masterInstance` name implies the type of the variable is `*Instance`

Reply:
Thanks Shlomi, working on this as well, appreciate the feedback. I'll post the new code with the corrections.

Reply:
I've pushed the changes, please let me know if this is ok. Thanks for all the help and input 😄  👍